### PR TITLE
adding screenshare types and tests

### PR DESCRIPTION
**Pull request by slorello89**
`screenshareType` parameter for start/set screenshare layout for opentok-java sdk

### CODE REVIEW

File: src/main/java/com/opentok/util/HttpClient.java [hunk 1]
```diff
@@ -591,13 +612,21 @@ public String setBroadcastLayout(String broadcastId, BroadcastProperties propert
         }
         String type = properties.layout().getType().toString();
         String stylesheet = properties.layout().getStylesheet();
+        String screenshareLayout = null;
         if(StringUtils.isEmpty(type)) {
-            throw new RequestException("Could not set the layout. Either an invalid JSON or an invalid layout options.");
+            throw new RequestException("Could not set the layout. Either an invalid JSON or an invalid layout " +
+                    "99options.");
```
Comment (dragonmantank) [inline src/main/java/com/opentok/util/HttpClient.java:618]: Typo, though really I would just revert this change, it doesn't really do much
```suggestion
                    "options.");
```

File: src/test/java/com/opentok/test/OpenTokTest.java [hunk 2]
```diff
@@ -8,32 +8,12 @@
...
 
 import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.client.ValueMatchingStrategy;
 import com.github.tomakehurst.wiremock.client.WireMock;
 import com.github.tomakehurst.wiremock.core.WireMockConfiguration;
 import com.github.tomakehurst.wiremock.junit.WireMockRule;
-import com.opentok.Archive;
+import com.opentok.*;
 import com.opentok.Archive.OutputMode;
-import com.opentok.ArchiveLayout;
```
Comment (dragonmantank) [inline src/test/java/com/opentok/test/OpenTokTest.java:16]: Saw this in other files... did we just have a bunch of extraneous imports?

**Review by dragonmantank — CHANGES_REQUESTED** [review]

File: src/main/java/com/opentok/util/HttpClient.java (hunk 1 above)
Comment (slorello89) [reply src/main/java/com/opentok/util/HttpClient.java]: Looks like I accidentally added it.

**Review by kellyjandrews — APPROVED** [review]
lgtm

**Review by dragonmantank — APPROVED** [review]
LGTM as well
